CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Changed `ui_tabs` and `ui_tab_button` to follow accessibility guidelines. As a side effect the gap between the tabs is now a little smaller.
```

Comment:
Maybe worth mentioning what you wrote in [the issue](https://github.com/bitcrowd/bitstyles_phoenix/issues/137)

> replace the `<ul>` with a `<div role="tablist">`